bindings/python/test/test_mongocrypt.py

```diff
@@ -265,8 +266,8 @@ def test_encrypt(self):
             self.assertEqual(ctx.state, lib.MONGOCRYPT_CTX_READY)
 
             encrypted = ctx.finish()
-            self.assertEqual(
-                BSON(encrypted).decode(), json_data('encrypted-command.json'))
+            self.assertEqual(bson.decode(encrypted, OPTS),
```

Comment:
bson.decode and bson.encode are preferred over using the BSON class (see PYTHON-1785).